CourseGrab/Network/Response.swift

```diff
@@ -21,9 +21,28 @@ extension Response {
     init(from decoder: Decoder) throws {
         let values = try decoder.container(keyedBy: CodingKeys.self)
         let rawDate = try values.decode(TimeInterval.self, forKey: .timestamp)
+        data = try values.decode(T.self, forKey: .data)
+        success = try values.decode(Bool.self, forKey: .success)
         timestamp = Date(timeIntervalSince1970: rawDate)
+    }
+    
+}
+
+struct DeviceTokenResponse: Codable {
+
+    let success: Bool
+    let timestamp: Date
+    
+}
```

Comment:
if there are any other endpoints have only have these two fields, we should rename this to like `SuccessResponse` or something so that we can reuse it. if not, then nvm